bin/templates/cordova/lib/plugin-build.gradle

```diff
@@ -21,24 +21,15 @@
 buildscript {
     repositories {
         mavenCentral()
+        jcenter()
     }
 
     // Switch the Android Gradle plugin version requirement depending on the
     // installed version of Gradle. This dependency is documented at
     // http://tools.android.com/tech-docs/new-build-system/version-compatibility
     // and https://issues.apache.org/jira/browse/CB-8143
-    if (gradle.gradleVersion >= "2.2") {
-        dependencies {
-            classpath 'com.android.tools.build:gradle:1.0.0+'
-        }
-    } else if (gradle.gradleVersion >= "2.1") {
-        dependencies {
-            classpath 'com.android.tools.build:gradle:0.14.0+'
-        }
-    } else {
-        dependencies {
-            classpath 'com.android.tools.build:gradle:0.12.0+'
-        }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.0.0+'
```

Comment:
I'm not very familiar with gradle compatibility issues, but it appears that this change removes backwards compatibility.  I'm guessing anything <1.0.0 is deprecated/EOL and no switching logic is required anymore.

It seems like the preceding comment should be updated or removed since there is no more switching logic and the information in CB-8143 is obsolete?  I don't know enough to update the comment appropriately, sorry.